### DISCUSSION TITLE
Docker build failed for v2.3.4

### DISCUSSION THREAD

**Discussion by FuzzyMistborn** [photoview/photoview#378] [General]
FYI @viktorstrate looks like the build for v 2.3.4 failed, not seeing a new image on Docker to pull.

https://github.com/photoview/photoview/actions/runs/863051808

**Comment by viktorstrate** [comment]
Thank you for the heads up, a small bug caused the build process to halt.
I've fixed it and the 2.3.4 update should be live now!
